src/components/catalog-filter/catalog-filter.controller.ts

```diff
@@ -91,6 +98,29 @@ export class CatalogFilterController implements angular.IController {
     });
   };
 
+  private keywordFilterExists = (keyword: string) => {
+      return _.some(this.ctrl.keywordFilter.values, (existingKeyword: string) => {
+          return keyword.toLowerCase() === existingKeyword.toLowerCase();
+      });
+  };
+
+  private highlightKeywordFilter = (keyword: string) => {
+      let filterTags: any = document.querySelectorAll('.active-filter.label.label-info.single-label');
+
+      let existingKeyword: any = _.find(filterTags, (tag: any) => {
+        return tag.innerText.trim() === ("Keyword: " + keyword.toLowerCase());
+      });
+
+      if (existingKeyword) {
+          this.$timeout(() => {
+              existingKeyword.style.opacity = 0;
```

Comment:
We should probably add and remove classes here to trigger the animation rather than manipulating the style object directly in a manner similar to the way Angular does.

Reply:
Hi, changed to add/remove animation class.